docs/src/references/mappers/flows.md

```diff
@@ -134,6 +132,82 @@ steps = [
 ]
 ```
 
+### Input connectors
+
+Messages can be consumed from MQTT, files and background processes.
+
+An MQTT connector is simply defined by a list of MQTT topics
+
+```toml
+# A flow subscribing to all measurement values and meta-data
+input.mqtt.topics = ["te/+/+/+/+/m/+", "te/+/+/+/+/m/+/meta"]
+```
+
+Messages can also be consumed from a file, each line being interpreted as the payload of a message
+which topic is by default the file path.
+
+```toml
+# A flow consuming log entries
+input.file.path = "/var/log/some-app.log"
+```
+
+By default, this file is followed as done by `tail -F`, waiting for new lines to be appended and consumed as messages.
+This behavior can be changed, the file being then read at regular intervals.
+
+```toml
+[input.file]
+path = "/var/log/some-app.log"
+topic = "some-app-log"
+interval = "1h"
+```
+
+Last but not least, messages can be consumed from a command output,
+each line being wrapped into a message which topic is the command line or a configured topic name.
+
+```toml
+# A flow subscribing to journalctl entries for the agent
+[input.process]
+topic = "tedge-agent-journalctl"
+command = "sudo journalctl --no-pager --follow --unit tedge-agent"
+```
+
+As for files, the command output can instead be consumed at regular intervals.
+
+```toml
+# A flow publishing new journalctl entries for the agent every hour 
+[input.process]
+topic = "tedge-agent-journalctl"
+command = "journalctl --no-pager --cursor-file=/tmp/tedge-agent-cursor --unit tedge-agent"
+interval = "1h"
+```
+
+### Output connectors
+
+Transformed messages and errors can be published over MQTT or appended to files.
+
+The default is to publish the transformed messages over MQTT on the topics specified by each message.
+And to direct all the errors to a specific topic, the `te/error` topic.
+
+```toml
+[output.mqtt]
+
```

Comment:
```suggestion
```
Unless that empty table had some relevance.

Reply:
Indeed not required. I only added it for symmetry.